pyscitt/setup.py

```diff
@@ -17,7 +17,8 @@
         "cryptography==38.*",  # needs to match ccf
         "httpx",
         "cbor2",
-        "pycose>=1.0.1",
+        # TODO: remove this once pycose >= 1.0.2 is released
+        "pycose @ git+https://github.com/TimothyClaeys/pycose@94db358eda640966c0e0e9148110b6c66763f9e5#egg=pycose",
```

Comment:
There's an xfail marker on `test_encoding.py`'s `test_verify_receipt` that can be removed now.

Reply:
Removed in https://github.com/microsoft/scitt-ccf-ledger/pull/90